src/bom/build.gradle.kts

```diff
@@ -77,6 +77,8 @@ dependencies {
         apiv("commons-lang:commons-lang")
         apiv("commons-net:commons-net")
         apiv("dnsjava:dnsjava")
+	apiv("io.burt:jmespath-core")
```

Comment:
Please refrain from using tabs.
We should probably setup a checkstyle verification